src/aspire/utils/BlockDiagonal.pyx

```diff
@@ -44,11 +41,8 @@ class BlockDiagonal:
             raise KeyError("Key {} is outside reversed nblocks {}".format(key, self.nblocks))
         return
 
-    def __copy__(self):
```

Comment:
This is an important function. Probably we should implement it.

Reply:
it is ? You are still looking at old commits.